tests/misc/tests/misc/misc.ts

```diff
@@ -1195,31 +1195,31 @@ const miscTest = (
 
       it("associated_token constraints (no init) - Can make with associated_token::token_program", async () => {
```

Comment:
`associated_token::token_program` will still fail with Token 2022 on non-init accounts. I've modified this test to use Token 2022 to show the failure(https://github.com/coral-xyz/anchor/pull/2603#issuecomment-1681257502).